packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md

```diff
@@ -308,6 +307,61 @@ function error(err) {
 }
 ```
 
+
+## Using with Okta SDKs
+
+Okta provides a number of SDKs that you might want to use the Sign-In Widget with, including Angular, React, and Vue.
+
+Using the Sign-In Widget with our SDKs that target the web is fairly straightforward.
+
+### Angular
+
+The [Okta Sign-In Widget and Angular guide](/code/angular/okta_angular_sign-in_widget/) shows the code you'll need in order to embed the Sign-In Widget in an Angular app.
+
+See the [Okta Angular + Custom Login Example](https://github.com/okta/samples-js-angular/tree/master/custom-login) for a working example.
+
+### React
+
+The [Okta Sign-In Widget and React guide](/code/react/okta_react_sign-in_widget/) shows the code you'll need in order to embed the Sign-In Widget in a React app.
+
+See the [Okta React + Custom Login Example](https://github.com/okta/samples-js-react/tree/master/custom-login) for a working example.
+
+### Vue
+
+<!--
+The [Okta Sign-In Widget and Vue guide](/code/vue/okta_vue_sign-in_widget/) shows the code you'll need in order to embed the Sign-In Widget in a Vue app.
+
+^^ todo: <https://github.com/okta/okta-developer-docs/issues/1440>
+-->
+
+See the [Okta Vue + Custom Login Example](https://github.com/okta/samples-js-vue/tree/master/custom-login) for a working example.
+
+### Mobile SDKs
+
+We also have mobile SDKs for Android, React Native, iOS, and Xamarin.
+
+For mobile apps, embedding the Sign-In Widget is not currently supported. A possible workaround is to redirect to Okta for authentication and [customize the hosted sign-in widget](/docs/guides/custom-hosted-signin/overview/). Support is provided, however, for building your own UI in mobile apps.
```

Comment:
This link looks broken: https://developer.okta.com/docs/guides/custom-hosted-signin/overview/